cmd/tetragon/flags.go

```diff
@@ -68,6 +68,8 @@ const (
 	keyRBSize      = "rb-size"
 	keyRBSizeTotal = "rb-size-total"
 
+	keyEventsQueueSize = "events-queue-size"
```

Comment:
Similar with previous comment, how about:
keyPerfEventsQueueSize = "perf-events-queue-size"

Reply:
ugh, did not notice how close those 2 options are now.. will rename

Reply:
so I changed it to follow --rb-* options, because it's closely related to ringbuffer, so now we have:

        keyRBSize      = "rb-size"
        keyRBSizeTotal = "rb-size-total"
       keyRBQueueSize = "rb-queue-size"

also I changed the new metrics names to follow the new name

Reply:
Sounds good, thanks!